app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -57,7 +57,8 @@ object UpperCaseBoolean {
 case class TextArea(
   constraint: TextConstraint,
   value: Expr,
-  displayWidth: DisplayWidth = DisplayWidth.DEFAULT
+  displayWidth: DisplayWidth = DisplayWidth.DEFAULT,
+  rows: Int = TextAreaRows.default
```

Comment:
Is it required to specify the default here?

Reply:
Yes. I have used default argument value at many places in tests.